### PR TITLE
new method for measuring disk space

### PR DESCRIPTION
**Pull request by mbiarnes**
new method to simply check the size of the disk (WORKSPACE and /home/jenkins)

https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1597
https://github.com/kiegroup/jenkins-pipeline-shared-libraries/pull/145

### CODE REVIEW

File: vars/util.groovy [hunk 1]
```diff
@@ -320,3 +320,14 @@ void cleanNode(String containerEngine = '') {
         cloud.cleanContainersAndImages(containerEngine)
     }
 }
+
+def spaceLeft() {
+    dir(env.WORKSPACE) {
```
Comment (gitgabrio) [inline vars/util.groovy:325]: HI @mbiarnes 
I'm not an expert: could you please explain me why that dir(env:WORKSPACE) is needed ?

Comment (mbiarnes) [reply vars/util.groovy:325]: as user jenkins you can't do in a job only things if you're in the WORKSPACE - I tested this and you see the whole machine, the /home/jenkins and the /hoem/jenkins/workspace - this work

**Review by gitgabrio — APPROVED** [review]
LGTM Thanks!

**Review by Ginxo — APPROVED** [review]
LGTM

**Pull request merged**